ddd/derdiedas.rb

```diff
@@ -0,0 +1,34 @@
+# TO DO
+# implement sets of words
+# rating for each word, how good remembered it was and when it was practiced last time
+# options - a batch or whole dictionary
+
+require_relative 'word'
+require_relative 'all_words'
+
+puts "Welcome! Let's test your knowledge of German articles\n"
+puts "Instructions:\n1-Play\n2-Add new words\n3-Write 'quit' if you want to quit.\n"
+print ">"
+
+option = gets.chomp
+
+case option
+when '1'
+  puts "Let's play."
+
+  guess = nil
+  loop do
+    word = @all_words[rand(0..@all_words.count-1)]
```

Comment:
this works, but this does the same and is simpler: `word = @all_words.sample`

Reply:
ah yes, before I had hash there and I couldn't use `sample`